h2o-core/src/main/java/water/api/FramesHandler.java

```diff
@@ -244,7 +244,8 @@ private FramesV3 doFetch(int version, FramesV3 s, boolean expensive) {
   public FramesV3 export(int version, FramesV3 s) {
     Frame fr = getFromDKV("key", s.frame_id.key());
     Log.info("ExportFiles processing (" + s.path + ")");
-    s.job = new JobV3(Frame.export(fr, s.path, s.frame_id.key().toString(), s.force, s.num_parts, s.compression));
+    Frame.CSVStreamParams csvParms = new Frame.CSVStreamParams().setSeparator(s.separator);
```

Comment:
can s.separator really be any char ? I guess yes, the user carries the consequences.

Reply:
yep, we just like our input API takes any character